README.md

```diff
@@ -17,3 +17,16 @@ Node: 18
    ```shell
    npm run wdio
    ```
+
+# Steps to install environment variable
```

Comment:
Introduce estos pasos directamente entre los pasos 1 y 2 de arriba en el orden:
1. Install dependencies
2. Copy .env.example file
3. Set up environment variables in .env file
4. Run tests